config/trainer_config.yaml

```diff
@@ -135,6 +135,7 @@ Tennis:
     buffer_size: 10240
     hidden_units: 256
     time_horizon: 1000
+    threaded: false
```

Comment:
Maybe you've already done this, but it might be good to try retraining with just the reset change and just disabling threading, so that we're not conflating the effects.

Reply:
I'm running both but just using this branch to do that. I won't merge until both results are in.